nifpga/fixedpointhelper.py

```diff
@@ -0,0 +1,59 @@
+from warnings import warn
+
+
+def bin_to_int(binary_string):
```

Comment:
This function doesn't seem to be used anywhere.

Reply:
Removed